CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [1.30](https://github.com/brave/brave-ios/releases/tag/v1.30)
+
+- Implemented History Sync. ([#3227](https://github.com/brave/brave-ios/issues/3227))
+- Implemented Brave News to display inline ads. ([#3872](https://github.com/brave/brave-ios/issues/3872))
+- Added "Delete All" functionality to clear history via history page. ([#3723](https://github.com/brave/brave-ios/issues/3723))
+- Added ability to share selected text from other apps to Brave via share sheet. ([#3863](https://github.com/brave/brave-ios/issues/3863))
+- Fixed an issue with rewards not being disabled when toggling off via rewards panel. ([#3591](https://github.com/brave/brave-ios/issues/
+- Fixed app theme issue at launch, being set to light mode, when Private Browsing only mode is enabled. ([#3936](https://github.com/brave/brave-ios/issues/3936))
+- Fixed menu to auto-close when a bookmark is selected. ([#3950](https://github.com/brave/brave-ios/issues/3950))
+- Fixed "Clear private data" global settings from resetting to defaults when menu is closed after changing the settings. ([#3958](https://github.com/brave/brave-ios/issues/3958))
+- Fixed crash when editing/deleting favorites when focus is set on URL(show more mode). ([#3974](https://github.com/brave/brave-ios/issues/3974))
```

Comment:
Space between `URL` & `(show more mode)`

Reply:
Show more mode is when URL is focused so not adding an & there.